### PR TITLE
Add support for mounting Kubernetes Secret for nats.creds file

### PR DESCRIPTION
**Pull request by joonas**
## Feature or Problem

We want the ability to either directly reference NATS jwt / seed, or have the ability to pass them in via Kubernetes Secret that gets volume mounted into the pod. This adds support for the volume mounting the secret.

### CODE REVIEW

**Review by protochron — APPROVED** [review]

File: charts/wadm/values.yaml [hunk 1]
```diff
@@ -14,12 +14,14 @@ wadm:
...
-      credsFile: ""
-      jetstreamDomain: ""
-      jwt: ""
-      seed: ""
       server: "127.0.0.1:4222"
+      jetstreamDomain: ""
       tlsCaFile: ""
+      creds:
+        jwt:
+        seed:
```
Comment (brooksmtownsend) [inline charts/wadm/values.yaml:22]: Are these keys of no value equivalent to specifying `""`

**Review by brooksmtownsend — COMMENTED** [review]
Just a question to make sure leaving JWT+Seed blank is fine, otherwise LGTM 👍🏻

File: charts/wadm/values.yaml (hunk 1 above)
Comment (joonas) [reply charts/wadm/values.yaml]: That's correct, and honestly there's no reason to have the double quotes missing here, so I went ahead and added those back for these values as well.

**Review by brooksmtownsend — DISMISSED** [review]
Test failed now? Weird. Good to go once resolved!

**Review by brooksmtownsend — APPROVED** [review]

**Pull request merged**